content/guides/advanced/verified-builds.md

```diff
@@ -0,0 +1,356 @@
+---
+date: 2024-09-26T00:00:00Z
+difficulty: intermediate
```

Comment:
lets make this featured to show at the top of the guides page
```suggestion
difficulty: intermediate
featured: true
```

Reply:
I would actually like to not put too much featuring on it until its actually ready. Wdyt? Still lots a work to do until its properly usable.

Reply:
that's fair then, we can ignore making it featured.

do you think we should generally wait until we merge this then?

Reply:
ship imo. Feature later. Get it indexed